thinc/model.py

```diff
@@ -35,7 +37,7 @@ class Model(Generic[InT, OutT]):
 
     global_id: int = 0
     global_id_lock: threading.Lock = threading.Lock()
-    _thread_local = create_thread_local({"operators": {}}, ModelThreadState)
```

Comment:
I had to add `ModelThreadState` as a workaround for thread-locals not being mutable. If that's not required here can you remove that class?